src/endpoints/seed/index.ts

```diff
@@ -280,16 +280,16 @@ export const seed = async ({
       rules: [
         {
           collections: ['roleAssignments'],
-          actions: ['create', 'update'],
+          actions: ['create', 'read', 'update'],
         },
       ],
     },
     {
       name: 'Contributor',
       rules: [
         {
-          collections: ['posts', 'pages', 'media'],
-          actions: ['create', 'update'],
+          collections: ['posts', 'pages', 'media', 'tenants'],
```

Comment:
Allowing mutating verbs on `tenants` would allow, for instance, the contributors to change the name of the AC or the websites associated with it - which is likely not what we would like. Do you know what specific permission over tenants was necessary? Perhaps just `read` would suffice?

Reply:
@stevekuznetsov Agreed. I updated the rules. But supposedly we have a global read on `tenants`.
>This is confusing because we are giving [tenant read access to all users](https://github.com/NWACus/web/blob/main/src/collections/Tenants/index.ts#L49-L51), so it must not be behaving how we intend it to be?

Do you think this is something we should look into?